tensorflow/tools/tf_sig_build_dockerfiles/Dockerfile.rocm

```diff
@@ -8,9 +8,15 @@ COPY setup.packages.rocm.cs7.sh setup.packages.rocm.cs7.sh
 COPY builder.packages.rocm.cs7.txt builder.packages.rocm.cs7.txt
 RUN /setup.packages.rocm.cs7.sh /builder.packages.rocm.cs7.txt
 
+ARG GPU_DEVICE_TARGETS="gfx900 gfx906 gfx908 gfx90a gfx1030"
+ENV GPU_DEVICE_TARGETS=${GPU_DEVICE_TARGETS}
+
 # Install ROCM
 ARG ROCM_VERSION
-ENV ROCM_PATH=/opt/rocm-${ROCM_VERSION}
+ARG INTERNAL_INSTALL
```

Comment:
Can we change "INTERNAL_INSTALL" to "CUSTOM_INSTALL"?

Reply:
Just update the variable name. Thanks.